docs/tutorials/runtime.md

```diff
@@ -0,0 +1,265 @@
+| `CheckpointHook`       | NORMAL (50)             |
+| `IterTimerHook`        | LOW (70)                |
+| `EvaluationHook`       | LOW (70)                |
+| `LoggerHook(s)`        | VERY_LOW (90)           |
+
+[sehedule strategy](./sehedule) has introduced how to modify `OptimizerHook`, `MomentumUpdaterHook` and `LrUpdaterHook`; `IterTimerHook` is used to record the time hook, currently does not support modification;
+
+Here we reveals how what we can do with `CheckpointHook`, `LoggerHooks`, and `EvaluationHook`.
+
+#### CheckpointHook
+
+The MMCV runner will use `checkpoint_config` to initialize [`CheckpointHook`](https://github.com/open-mmlab/mmcv/blob/9ecd6b0d5ff9d2172c49a182eaa669e9f27bb8e7/mmcv/runner/hooks/checkpoint.py).
+
+```python
+checkpoint_config = dict(interval=1)
+```
+
+The users could set `max_keep_ckpts` to only save only small number of checkpoints or decide whether to store state dict of optimizer by `save_optimizer`.
+More details of the arguments are [here](https://mmcv.readthedocs.io/en/latest/api.html#mmcv.runner.CheckpointHook)
+
+#### Log config
+
+The `log_config` wraps multiple logger hooks and enables to set intervals. Now MMCV supports `WandbLoggerHook`, `MlflowLoggerHook`, and `TensorboardLoggerHook`.
+The detail usages can be found in the [doc](https://mmcv.readthedocs.io/en/latest/api.html#mmcv.runner.LoggerHook).
+
+```python
+log_config = dict(
+    interval=50,
+    hooks=[
+        dict(type='TextLoggerHook'),
+        dict(type='TensorboardLoggerHook')
+    ])
+```
+
+#### Evaluation config
+
+The config of `evaluation` will be used to initialize the [`EvalHook`](https://github.com/open-mmlab/mmclassification/blob/master/mmcls/core/evaluation/eval_hooks.py).
+Except the key `interval`, other arguments such as `metrics` will be passed to the `dataset.evaluate()`
+
+```python
+evaluation = dict(interval=1, metric='accuracy', metric_options={'topk': (1, )})
+```
+
+You can also save the model weight when the best verification result is obtained by modifying the parameter `save_best`:
+
+```python
+evaluation = dict(interval=1, save_best=True, metric='accuracy', metric_options={'topk': (1, )})
+```
+
+When running some large experiments, you can skip the validation step at the beginning of training by modifying the parameter `start` to save time. as follows:
+
+```python
+evaluation = dict(interval=1, start=200, metric='accuracy', metric_options={'topk': (1, )})
+```
+
+Indicates that, in one epoch, only the training process is performed before the 200th epoch, withith validation process; starting from the 200th epoch, validation rocess would be executed after training process.
+
+```{note}
+1. In the default configuration files of MMClassification, the evaluation field is generally placed in the datasets configs.
+
+2. 'EvalHook' in 'MMClassification/mmcls/core/evaluation/eval_hooks.py' will be deprecated, recommend to use 'EvaluationHook' in MMCV as above.
```

Comment:
This note can be removed because config file modification is not relevant to use which implementation.